### PR TITLE
Modified existing and added new snippets for Kaluptic Psychosis (Schizophrenia)

### PR DESCRIPTION
**Pull request by Cataclysmanic**
#### Summary
Content "Modified existing snippets for the trait Kaluptic Psychosis and added new ones."

#### Purpose of change
The purpose is to add more character to the messages and snippets that the player will experience with Kaluptic Psychosis. Adding more depth to the player's character than just 'homicidal maniac who talks to themselves.' There's also a few grammatical errors and other more annoying messages to be seen.

#### Describe the solution
More casual, worried, disdainful and humourous lines will be added to deliver more personality to the snippets and the thoughts behind them. It'll make it clear that the character has loss of control over their thoughts but isn't just a murderhobo suffering from a mental illness. Older lines will be fixed and made more enjoyable to read by removing commas and semicolons where they are unnecessary and adding new ones where they are.

#### Testing
I went into the json file in my computer's directory and modified it with notepad++ before testing it on my installation of the game. I've gotten and fixed any syntax errors that the game brought up, and the new messages appear without issue. A suggestion for testing would be to open a world with Kaluptic Psychosis as a trait and waiting around for messages to appear.

### CODE REVIEW

File: data/json/snippets/schizophrenia.json [hunk 1]
```diff
@@ -4,41 +4,60 @@
...
+      "…and just where have you been?",
+      "You hungry?  I've got plenty of food.",
+      "You thirsty?  I've got water.",
       "Why would you do that?",
-      "Please, don't go.",
+      "Please don't go.",
       "Don't leave me alone!",
       "Yeah, sure.",
-      "No way, man.",
+      "No way man.",
```
Comment (Kamayana) [inline data/json/snippets/schizophrenia.json:23]: I'm pretty sure "No way, man" is actually the correct punctuation here.

Comment (Cataclysmanic) [reply data/json/snippets/schizophrenia.json:23]: Grammatically, and by the loose rules of English, they're both actually correct. However inn a conversation or self speech, a comma there would imply a pause. Which to me didn't sound quite right given the context.

File: data/json/snippets/schizophrenia.json [hunk 2]
```diff
@@ -49,43 +68,71 @@
...
-      "Hey fix me up.",
+      "Hey, fix me up will ya?",
       "I need healing!",
-      "I hurt all over…",
+      "It hurts all over…",
       "You can put me back together, right?",
       "I… I can't move my legs!",
       "Medic!",
+      "You gonna let me die too?!",
+      "No! I can't die now!",
```
Comment (Fris0uman) [inline data/json/snippets/schizophrenia.json:95]:
```suggestion
      "No!  I can't die now!",
```

File: data/json/snippets/schizophrenia.json [hunk 3]
```diff
@@ -4,41 +4,60 @@
     "category": "schizo_self_talk",
     "text": [
       "Hey, can you hear me?",
-      "Don't touch me.",
+      "I think I know what I'm doing.",
```
Comment (Maleclypse) [inline data/json/snippets/schizophrenia.json:7]:
```suggestion
      "I think I know what I'm doing.",
      "Don't touch me.",
```

File: data/json/snippets/schizophrenia.json [hunk 4]
```diff
@@ -49,43 +68,71 @@
       "Hey, you're bleeding.",
       "Your wound looks pretty bad.",
       "Shouldn't you put a bandage on that?",
-      "Please don't die!  No one else lets me kill things!",
-      "You look hurt; did I do that?",
+      "You look hurt; did I do that?  I'm sorry.",
```
Comment (Maleclypse) [inline data/json/snippets/schizophrenia.json:71]:
```suggestion
      "You look hurt; did I do that?  I'm sorry.",
      "Please don't die!  No one else lets me kill things!",
```

File: data/json/snippets/schizophrenia.json [hunk 5]
```diff
@@ -49,43 +68,71 @@
...
+      "You should take a shower.",
+      "We should go home, it's not safe here.",
+      "We're in over our heads!",
+      "There's something following us, I can feel it.",
       "I'm your best friend, right?",
-      "I love you!",
       "How are you today?",
-      "Do you think it will rain today?",
+      "Sorry about earlier, can you forgive me?",
+      "Do you think it'll rain today?",
```
Comment (Maleclypse) [inline data/json/snippets/schizophrenia.json:130]:
```suggestion
      "Do you think it'll rain today?",
      "I love you!",
```

**Pull request merged**